synapse/storage/monthly_active_users.py

```diff
@@ -165,19 +172,33 @@ def get_registered_reserved_users_count(self):
 
     @defer.inlineCallbacks
     def upsert_monthly_active_user(self, user_id):
+        """Updates or inserts monthly active user member
```

Comment:
I kinda wish I knew what this meant, but I guess that's not a new problem here :/